ext/opentelemetry-ext-flask/src/opentelemetry/ext/flask/__init__.py

```diff
@@ -150,6 +161,20 @@ def _teardown_flask_request(exc):
             context.detach(flask.request.environ.get(_ENVIRON_TOKEN))
 
 
+def _disable_trace(url):
+    excluded_hosts = configuration.Configuration().FLASK_EXCLUDED_HOSTS
+    excluded_paths = configuration.Configuration().FLASK_EXCLUDED_PATHS
```

Comment:
Is this separation necessary? I think we can just define a sequence of regular expressions in one environment variable and if the hostname or path matches any regular expression it is rejected. This will be useful for the end user who would not need to repeat a lot of related paths in the environment variable, but use a single regex that matches them all.

Reply:
I worry a little bit about that, as it makes it hard if you happen to have a host or path that share the same name.

Horrible example, but what about the following:

host: localhost
path: check_if_localhost

and you only want to filter for the localhost host.